node/src/components/linear_chain_sync/operations.rs

```diff
@@ -312,6 +276,73 @@ where
     Ok(None)
 }
 
+pub(super) struct SwitchBlockInfo {
+    validator_weights: BTreeMap<PublicKey, U512>,
+    era_start: Timestamp,
+    switch_block_height: u64,
```

Comment:
Maybe just `height`? (Seems redundant.)
```suggestion
    height: u64,
```

Reply:
Fixed in a7a3a3118